language-testutils/src/test/java/de/jplag/testutils/datacollector/TestData.java

```diff
@@ -0,0 +1,34 @@
+package de.jplag.testutils.datacollector;
+
+import java.io.IOException;
+import java.util.List;
+
+import de.jplag.Language;
+import de.jplag.ParsingException;
+import de.jplag.Token;
+
+/**
+ * Any kind of test source provider
```

Comment:
As this is a top-level interface, this could be more descriptive. A reader could ask: What kind of data is it? A set of submissions? A submission? A file? Code? XML?